LenovoLegionToolkit.WPF/Windows/Automation/CreateAutomationPipelineWindow.xaml.cs

```diff
@@ -40,7 +40,8 @@ public partial class CreateAutomationPipelineWindow
         new WiFiConnectedAutomationPipelineTrigger(Array.Empty<string>()),
         new WiFiDisconnectedAutomationPipelineTrigger(),
         new TimeAutomationPipelineTrigger(false, false, TimeExtensions.UtcNow, Enum.GetValues<DayOfWeek>()),
-        new OnStartupAutomationPipelineTrigger()
+        new OnStartupAutomationPipelineTrigger(),
+        new PeriodicAutomationPipelineTrigger(null)
```

Comment:
This array also defines order on the UI (I know, not the most obvious and best design), but I would place it one row up, so it appears next to "At specified time" trigger on the list.